components/tabs/index.tsx

```diff
@@ -153,6 +160,7 @@ export default class Tabs extends React.Component<TabsProps, any> {
 
     const renderTabBar = () => (
       <ScrollableInkTabBar
+        inkBarAnimated={inkBarAnimated}
```

Comment:
the [rc/tabs documentation](https://github.com/react-component/tabs#libscrollableinktabbar) does not mention about `inkBarAnimated` props, but i think the documentation misses it(may be I can bring up another pr to rc/tabs ? @yiminghe ). 

I read the [source code](https://github.com/react-component/tabs/blob/master/src/InkTabBarMixin.jsx#L89), and I think it's a stable feature.